source/openapi-admin-v3.yaml

```diff
@@ -5525,7 +5547,71 @@ components:
         error_code:
           type: string
           description: The error type.
-    AtlasCluster:
+    AtlasConfigResponse:
+      type: object
+      properties:
+        config:
+          type: object
+          properties:
+            clusterId:
+              type: number
+              example: 05e821b3975d271289f372e3a
+              description: The service ID.
+            clusterName:
+              type: string
+              example: Cluster0
+              description: |-
+                The service name used to refer to the data source within
+                this Atlas app. The name may be at most 64 characters
+                long and must only contain ASCII letters, numbers,
+                underscores, and hyphens.
+            clusterType:
+              type: string
+              example: atlas
+              description: The type of service.
+            flexible_sync:
+              $ref: "#/components/schemas/FlexibleSync"
+            groupName:
+              type: string
+              example: Project 0
+              description: The name of the group that contains the service.
+            orgName:
+              type: string
+              example: MongoDB_Org
+              description: The name of the organization that contains the service.
+            readPreference:
+              type: string
+              enum:
+                - primary
+                - primaryPreferred
+                - secondary
+                - secondaryPreferred
+                - nearest
+              description: >-
+                The [read preference](https://www.mongodb.com/docs/atlas/app-services/mongodb/read-preference/)
+                mode for read requests to the data source.
+            wireProtocolEnabled:
+              type: boolean
+              description: >-
+                If true, clients may [connect to the app over the
+                MongoDB Wire Protocol](https://www.mongodb.com/docs/atlas/app-services/mongodb/wire-protocol/#connect-over-the-wire-protocol).
+        secret_config:
+          type: object
+          description: "[Deprecated] The configuration for a third-party service."
+    ServiceResponse:
+      type: object
+      properties:
+        _id: string;
+        name: string;
+        type: string;
+        version: number;
+        last_modified: number;
+    ReadPreferencesTagSet:
+      type: string
+      enum:
+        - server1
+        - server2
+    CreateServiceRequest:
```

Comment:
This seems to be specifically for data sources - can we use a less generic name? Also maybe an opportunity to reuse the `AtlasCluster` schema?